uk.co.compucorp.civicrm.tasksassignments/views/modal/task.html

```diff
@@ -74,14 +75,14 @@
             </div>
             <!-- Subject -->
             <div class="row">
-                <div class="col-xs-12 col-sm-6" ng-show="!showFieldSubject">
+                <div class="col-xs-12 col-sm-6" ng-show="!task.subject && !showFieldSubject">
                     <span class="form-control-static">
                         <a href class="btn btn-indent" ng-click="showFieldSubject = true;">
```

Comment:
That's AngularJS preferred syntax, you don't give `href` a value and it knows that it's a link that shouldn't do anything if clicked